src/dmangle.d

```diff
@@ -630,17 +629,17 @@ public:
          * -0X1.1BC18BA997B95P+79   => N11BC18BA997B95P79
          * 0X1.9P+2                 => 19P2
          */
-        if (Port.isNan(value))
+        if (isNaN(value))
```

Comment:
No problem - would you prefer the fully qualified name or putting these functions as static functions into a helper struct (analogous to `Port`)?